lib/logstash/environment.rb

```diff
@@ -114,6 +95,43 @@ def bundler_setup!(options = {})
       ::Bundler.setup
     end
 
+    def runtime_jars_root(dir_name, package)
+      ::File.join(dir_name, package, "runtime-jars")
+    end
+
+    def test_jars_root(dir_name, package)
+      ::File.join(dir_name, package, "test-jars")
+    end
+
+    def load_runtime_jars!(dir_name="vendor", package="jar-dependencies")
+      load_jars!("#{runtime_jars_root(dir_name, package)}/*.jar")
+    end
+
+    def load_test_jars!(dir_name="vendor", package="jar-dependencies")
+      load_jars!("#{test_jars_root(dir_name, package)}/*.jar")
```

Comment:
`File::SEPARATOR` is the constant for the platform specific stuff, but `File.join` doesnt care if the path exist or not.